docs/content/specs-defs/includes/bicep/shared/non-functional/BCPNFR1.md

```diff
@@ -20,7 +20,7 @@ tags: [
 priority: 11010
 ---
 
-#### ID: BCPNFR1 - Category: Inputs - Data Types
+#### ID: BCPNFR1 - Category: Inputs - User-defined types - General
```

Comment:
Hey @matebarabas,
thanks for the comment. The spec is actually absolutely here to be reviewed - but - is unfortunately dependent on [this](https://github.com/Azure/Azure-Verified-Modules/pull/1737) PR which is pending a review by @eriqua :)
In any case, you have a point - yet I'd argue the title should be changed to make more sense in the context of the other specs (which are about specific topics in the space of UDTs). 
Also, sure this could break references (if they happen to exist), but personally I am strongly against prioritizing backward compatibility over usability/correctness. Just to explain the angle I'm coming from 😉

Reply:
So... can this comment be resolved, @matebarabas?

Reply:
Can any applicable link be updated via search and replace?

Reply:
> Can any applicable link be updated via search and replace?

No, as the ones I was referring to are auto-generated based on the title. I agree to Alex's comment on not prioritizing backward compatibility if we feel something is just simply off. Feel free to resolve this.

> So... can this comment be resolved, @matebarabas?

Yes please, that's what I was hoping to express by sending a thumb-up on your comment. :) I knew I should've been more explicit. :)